### DISCUSSION TITLE
Monailabel train from scratch

### DISCUSSION THREAD

**Discussion by RotoDrache** [Project-MONAI/MONAILabel#1276] [Q&A] [answered]
This is a basic question but I couldn't find the answer.
When you are training from scratch can you stop or interrupt the training process and pick up where you left it at a later date? If so is it best to wait until the current series of epochs is finished?
Also will training in this fashion affect the quality of your final model. I also assume that you start with the next image in the sequence.
Many thanks inadvance

**Comment by tangy5** [comment, marked as answer]
You can stop the training process, MONAI Label plugin should have a button for that. But I feel your question is how to save an inter-medium model for future use.  Now the plugin have an "option" panel that can chose the epochs for training. You can train smaller epochs and save the model, or train it again when new labeled data are annotated. 

Typically, train smaller set and more iterations should be better, and it would be better each iteration new annotated data can be added. This interactive process can achieve better results.

**Reply by RotoDrache** [reply]
Thank-you that actually answered my questions
